pallets/capacity/src/migration/v3.rs

```diff
@@ -0,0 +1,130 @@
+use crate::{BalanceOf, BlockNumberFor, Config, FreezeReason, Pallet, StakingType};
+use frame_support::{
+	pallet_prelude::{GetStorageVersion, IsType, Weight},
+	traits::{
+		fungible::MutateFreeze, Get, LockIdentifier, LockableCurrency, OnRuntimeUpgrade,
+		StorageVersion,
+	},
+};
+use parity_scale_codec::Encode;
+use sp_core::hexdisplay::HexDisplay;
+
+const LOG_TARGET: &str = "runtime::capacity";
+
+#[cfg(feature = "try-runtime")]
+use sp_std::{fmt::Debug, vec::Vec};
+
+const STAKING_ID: LockIdentifier = *b"netstkng";
+
+/// Only contains V2 storage format
+pub mod v2 {
+	use super::*;
+	use frame_support::{storage_alias, Twox64Concat};
+	use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+	use scale_info::TypeInfo;
+
+	#[derive(Encode, Decode, PartialEq, Debug, TypeInfo, Eq, MaxEncodedLen)]
+	/// The StakingDetails struct
+	pub struct StakingDetails<T: Config> {
+		/// The amount a Staker has staked, minus the sum of all tokens in `unlocking`.
+		pub active: BalanceOf<T>,
+		/// The type of staking for this staking account
+		pub staking_type: StakingType,
+	}
+
+	#[storage_alias]
+	/// alias to StakingAccountLedger storage
+	pub(crate) type StakingAccountLedger<T: Config> = StorageMap<
+		Pallet<T>,
+		Twox64Concat,
+		<T as frame_system::Config>::AccountId,
+		StakingDetails<T>,
+	>;
```

Comment:
I've lost track--is this before or after @shannonwells staking ledger changes for PB?

Reply:
> I've lost track--is this before or after @shannonwells staking ledger changes for PB?

Currency->fungible is after the staking ledger changes.